pennylane/qchem/__init__.py

```diff
@@ -58,7 +58,14 @@
 from .observable_hf import fermionic_observable, qubit_observable, jordan_wigner
 from .number import particle_number
 from .spin import spin2, spinz
-from .structure import read_structure, active_space, excitations, hf_state, excitations_to_wires
+from .structure import (
+    read_structure,
+    active_space,
+    excitations,
+    hf_state,
+    excitations_to_wires,
+    mol_data_from_pubchem,

```

Comment:
We can keep this function in `qchem`, but I think `data` module could be also an option:

`qml.data.pubchem('H2O')`